service/src/main/java/bio/terra/cbas/util/methods/GithubUrlDetailsManager.java

```diff
@@ -0,0 +1,84 @@
+package bio.terra.cbas.util.methods;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Why does this need to be a component? It looks more like a utility function class?

Reply:
+1. I think `extractDetailsFromUrl` can just be a utility function inside a `GithubUtils` class or existing [MethodUtils class](https://github.com/DataBiosphere/cbas/blob/main/service/src/main/java/bio/terra/cbas/common/MethodUtil.java).